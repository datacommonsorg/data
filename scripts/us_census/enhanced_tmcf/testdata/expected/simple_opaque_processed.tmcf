Node: E:simple_opaque_processed->E0
typeOf: schema:Place
dcid: C:simple_opaque_processed->GEO_ID
name: C:simple_opaque_processed->NAME

Node: E:simple_opaque_processed->E1
typeOf: schema:StatVarObservation
name: Number of establishments
value: C:simple_opaque_processed->ESTAB
variableMeasured: C:simple_opaque_processed->new_col_0
observationAbout: E:simple_opaque_processed->E0
observationDate: C:simple_opaque_processed->YEAR
