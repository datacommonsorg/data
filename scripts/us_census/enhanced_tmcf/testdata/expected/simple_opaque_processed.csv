GEO_ID,NAICS2012,TAXSTAT,TYPOP,YEAR,ESTAB,new_col_0
country/USA,21,0,0,2012,25417,data.census.gov/table/EC1200A1/variable/ESTAB__naics2012--21++taxstat--0++typop--0
country/USA,22,0,0,2012,17595,data.census.gov/table/EC1200A1/variable/ESTAB__naics2012--22++taxstat--0++typop--0
country/USA,23,0,0,2012,598065,data.census.gov/table/EC1200A1/variable/ESTAB__naics2012--23++taxstat--0++typop--0
